requirements/edx/development_layer.in

```diff
@@ -9,9 +9,13 @@
 # update scripts/dependencies/development.txt accordingly.
 
 -c ../constraints.txt
+-c pip-tools.txt                        # pip-tools and its dependencies, for managing requirements files
```

Comment:
Note for reviewers.
While working on this PR, it was uncovered to me that we cannot constraint based off of a requirements file which itself references other requirements files. In case I used `-c testing.txt` here it would end up constraining requirements according to `testing.txt` but also fetch all the requirements from `testing.txt` explicitly into the `development_layer.txt` effectively repeating the requirements in multiple files and negating all the effort that we are doing in this ticket.